charts/federation-v2/charts/controllermanager/templates/clusterrole.yaml

```diff
@@ -13,35 +13,57 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
+  - get
+  - watch
+  - list
+  - update
 - apiGroups:
   - multiclusterdns.federation.k8s.io
   resources:
   - '*'
   verbs:
-  - '*'
```

Comment:
nit: If all verbs are allowed, why not use `*`?

Reply:
a full supported kube verb list should be `get, list, watch, create, update, patch, delete, deletecollection, and proxy`. So it is different.